recursio-client/UI/Menus/StartMenu.gd

```diff
@@ -80,7 +95,18 @@ func _ready():
 	get_tree().get_root().call_deferred("add_child", new_scene)
 
 
+func _process(_delta):
+	if not _in_game:
+		return
+	
+	if not Input.is_action_just_pressed("ui_cancel"):
+		return
```

Comment:
This seems like a somewhat convoluted way to do it :sweat_smile: why not just react to `ui_cancel` for making it invisible?

Reply:
Done